Src/DependencyCollector/Shared/Implementation/HttpDesktopDiagnosticSourceListener.cs

```diff
@@ -52,8 +53,10 @@ public void OnNext(KeyValuePair<string, object> value)
                     // event was temporarily introduced in DiagnosticSource and removed before stable release
                     case "System.Net.Http.Request": 
                     {
+                        // request is never null
                         var request = (HttpWebRequest)this.requestFetcherRequestEvent.Fetch(value.Value);
-                        this.httpProcessingFramework.OnRequestSend(request);
+                        DependencyCollectorEventSource.Log.BeginCallbackCalled(request.GetHashCode(), value.Key);
```

Comment:
Please use a different log message. I understand there is a task opened for fixing the calls backs, but having a new callback log message for 3-4 switch cases in DiagnosticSourceListener should be very easy fix.